docs/troubleshooting/updating-nitric.mdx

```diff
@@ -0,0 +1,18 @@
+## Updating Nitric
+
+If you're ready to upgrade your projects to a newer version of Nitric
+
+- Start by refreshing any versions of the membrane which are currently residing on Docker:
+  - Run `docker image rm nitricimages/membrane-local`
+  - Checkout the latest version of the membrane
+  - Run `make dev-docker` (or other target e.g. `make aws-docker`. Please refer to the [membrane readme](https://github.com/nitrictech/membrane/blob/main/README.md) for more details
+  - Update the local membrane with `docker pull nitricimages/membrane-local`
+  - Note - If you encounter issues with the previous step(s) then prune docker with the following command `docker system prune -a`
+- Update your local store of templates `nitric templates:repos:add`
+- In your project(s) remove the templates folder which resides in .nitric
+- Create a new project for each of the templates which you are using `nitric make:service` this will update the .nitric folder with the appropriate templates.
```

Comment:
This could/should also be done by the `nitric doctor` command or perhaps a new command like `nitric repair`, which finds services in the nitric.yaml file and if the referenced `runtime` is missing in the local `.nitric` file, prompts to add them back from the template repo.

So then the steps in the docs would be:

1. delete `./nitric/templates`
2. run `nitric repair` and follow the prompts

Reply:
@tjholm thoughts?

Reply:
Adding this to CLI ticket.